packages/framework/src/Form/Admin/Complaint/Status/ComplaintItemsType.php

```
@@ -16,6 +15,8 @@
     #[Override]
     public function getParent(): string
     {
-        return CollectionType::class;
+        $resolver->setDefaults([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resolver'. (undefined)

(UnusedLocalVariable)